server/Pipfile

```diff
@@ -9,4 +9,4 @@ verify_ssl = true
 flask = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.8"
```

Comment:
You'll want to check with the other team members if it's okay to change the python version to 3.8